components/led_strip/inc/led_strip/led_strip.h

```diff
@@ -1,86 +1,73 @@
-/*  ----------------------------------------------------------------------------
+/*  ---------------------------------------------------------------------------
     File: led_strip.h
     Author(s):  Lucas Bruder <LBruder@me.com>
     Date Created: 11/23/2016
-    Last modified: 11/25/2016
+    Last modified: 11/26/2016
 
-    Description: LED Library for driving various led strips
-    
-    ------------------------------------------------------------------------- */
+    Description: 
+    This library can drive led strips through the RMT module on the ESP32.
+    ------------------------------------------------------------------------ */
 
 #ifndef LED_STRIP_H
 #define LED_STRIP_H
 
+#include <driver/rmt.h>
 #include <driver/gpio.h>
 #include "freertos/FreeRTOS.h"
-#include "freertos/task.h"
 
-#include <stdbool.h>
-#include <stdint.h>
 #include <stddef.h>
 
-#define LED_STRIP_REFRESH_PERIOD_MS 30
+enum led_strip_type_t {
```

Comment:
change back to rgb_led_type to highlight the differences in individual rgb leds, not led stirps